src/components/controller/controller.ts

```diff
@@ -1,9 +1,14 @@
 import AppLoader from './appLoader';
-import {Endpoints} from '../enums'
+import {Endpoints, Languages} from '../enums'
```

Comment:
There are different code styles. I mean sometimes `import { Value } from './library'`, sometimes `import {Value} from './library'`. Please use formatting

Reply:
Brought to a unified style with whitespaces